packages/bip44/src/index.ts

```diff
@@ -54,7 +81,7 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
       if (!approved) {
```

Comment:
Since we are touching this anyway can we hexlify the pub key in the confirmation?